Splitter : add test for resizing multiple panes

jdvictoria opened the pull request:
Added the following for specific scenarios:

1. Multiple Items in DataSource
2. Resize Behavior with Multiple Items
3. Resize Handles Visibility
4. Test for Resizing Right and Left
5. Test for Percentage Resizing

jdvictoria committed:
Add Unit Tests

Added the following for specific scenarios:

1. Multiple Items in DataSource
2. Resize Behavior with Multiple Items
3. Resize Handles Visibility
4. Test for Resizing Right and Left
5. Test for Percentage Resizing

jdvictoria committed:
Remove duplicate test

jdvictoria committed:
Fix Unit Test for No Mouse Button Release

jdvictoria committed:
Fix Values

jdvictoria committed:
Refactor tabPanel for Checks

jdvictoria committed:
Revert

AlexanderMoiseev commented on packages/devextreme/testing/tests/DevExpress.ui.widgets/tabPanel.width.tests.js:
```diff
@@ -66,10 +66,10 @@ QUnit.module('Tabs width', () => {
 
             if(options.width > 250) {
                 this.assert.strictEqual(getOuterWidth(this._getTabItem(0)) <= 140, true, getOuterWidth(this._getTabItem(0)) + ' <= 140');
-                this.assert.strictEqual(getOuterWidth(this._getTabItem(1)) <= 160, true, getOuterWidth(this._getTabItem(1)) + ' <= 160');
```
why did you change tabPanel tests?

AlexanderMoiseev commented on packages/devextreme/testing/tests/DevExpress.ui.widgets/splitter.tests.js:
```diff
@@ -168,6 +168,61 @@ QUnit.module('Resizing', moduleConfig, () => {
+            assertLayout(items, ['38.4146', '28.252', '33.3333'], assert);
+        });
+
+        QUnit.test('resize handles visibility with multiple items in dataSource', function(assert) {
```
i believe we have these kind of tests already

jdvictoria committed:
Remove duplicate test

jdvictoria committed:
Revert tabPanel changes

jdvictoria committed:
Update splitter.tests.js

Resolve:

"Your second test
'percentage resizing during drag right and left' also can use the same code as the first one, you can just pass different resizeDistance parameter"

jdvictoria committed:
single resizeHandle test

jdvictoria committed:
Update splitter.tests.js

AlexanderMoiseev committed:
parameters for test added, redundant tests removed

AlexanderMoiseev committed:
clean code

AlexanderMoiseev commented:
change the title please according to our style guide
